### ISSUE TITLE
Alert Authentication Method not working properly

### ISSUE THREAD

**Issue by fertaku** [SeleniumHQ/selenium#5471]
## Meta -
OS:  Windows 7

Selenium Version:  3.4.3

Browser:  Chrome

Browser Version:  Version 64.0.3282.140 (Offizieller Build) (64-Bit)

## Expected Behavior -
Alert Authenticate method authenticates and execution continues

## Actual Behavior -
Nothing happens. Authenticate method freezes. I think around this point (webdriver.py:295)

```
        if response:
            self.error_handler.check_response(response)
            response['value'] = self._unwrap_value(
                response.get('value', None))
            return response
```

## Steps to reproduce -

This is the function i'm using to handle the alert:

```
    def login_browser_alert(driver):
        WebDriverWait(driver, 3).until(EC.alert_is_present())
        current_window = driver.window_handles[0]
        driver.switch_to.alert.authenticate('admin', 'admin')
        driver.switch_to.window(current_window)
```

This is how I navigate to the page:

```
    def goto_page(self):
            self.driver.get(self.URL)
            login_browser_alert(self.driver)
```

I'm using http://the-internet.herokuapp.com/basic_auth as a dummy Authentication example. Credentials are user: "admin" pass: "admin"

**Comment by lmtierney** [comment]
This was actually an experimental feature back in the day and I believe none of the current webdrivers support authentication alerts.
Some documentation should probably be added to the python docstring to indicate this as is done with Java. I'll leave this open as a task to do so.

**Comment by p0deje** [comment]
None of the drivers currently support this. I guess python bindings should deprecate this. Ruby already removed :)

**Comment by lmtierney** [comment]
The method has been removed as it has been so in other languages for quite some time. Closed via  95fdcb0
